core/templates/pages/contributor-dashboard-page/opportunities-list-item/opportunities-list-item.component.html

```diff
@@ -7,6 +7,15 @@
            class="progress">
         <div class="progress-bar bg-success"
              role="progressbar"
+             [style.width.%]="progressValue"
+             [attr.aria-valuenow]="progressValue !== undefined ? progressValue.toString() : ''"
+             [attr.aria-valuemin]="0"
+             [attr.aria-valuemax]="100"
+             [attr.aria-label]="progressBarLabel"
+             title="progressBar"
+             ngbTooltip="{{opportunity?.translationsCount}}/{{opportunity?.totalCount}} cards accepted"
+             container="body"
+             placement="bottom"
```

Comment:
Don't think most of these attributes are necessary, just a `aria-label` tag here and in the 3 types of progress bars below.

Reply:
Noted!

Reply:
will resolve and request for review